stdlib.sh

```diff
@@ -136,7 +150,11 @@ EOC
 # Externally set control-variables:
 #
 # STDLIB_WANT_MEMCACHED	- Load native memcached functions;
-# STDLIB_API		- Specify the stdlib API to adhere to.
+# STDLIB_API		- Specify the stdlib API to adhere to;
+# STDLIB_COLOUR_MAP - Specify custom colour map file, default being
+#                     '/usr/local/lib/stdlib.colours';
```

Comment:
`stdlib.colours` should really live in `/etc/stdlib.colours`, `/etc/stdlib/colours`, or `/usr/local/etc/stdlib.colours`...